src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -12,5 +12,89 @@
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="console" />
         </service>
+
+        <service id="console.command.about" class="Symfony\Bundle\FrameworkBundle\Command\AboutCommand">
```

Comment:
Tend to agree, however it's not done in webserverbundle/securitybundle. So i wasnt sure..  but yeah framework/twigbundle could do it as of now.